packages/graphql-yoga/__tests__/integration.spec.ts

```diff
@@ -7,13 +7,16 @@ const yoga = new GraphQLServer({ schema, enableLogging: false })
 
 describe('Requests', () => {
   it('should send introspection query', async () => {
-    const response = await yoga.inject({
+    const response = await yoga.inject<IntrospectionQuery>({
       document: getIntrospectionQuery(),
     })
 
-    expect(response.statusCode).toBe(200)
-    expect(response.errors).toBeUndefined()
-    expect(response.data.__schema.queryType.name).toBe('Query')
+    expect(response.status).toBe(200)
+
+    const responseBody = await response.json();
```

Comment:
This util is only used for testing purposes so should we just send a JSON response with data and errors?